scripts/bash/enceladus_env.template.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+# Copyright 2018-2019 ABSA Group Limited
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Environment configuration
+MENAS_URI="http://localhost/menas/api"
+STD_HDFS_PATH="/bigdata/std/std-{0}-{1}-{2}-{3}"
+SPLINE_MONGODB_URL="mongodb://localhost:27017"
+SPLINE_MONGODB_NAME="spline"
+
+export SPARK_HOME="/opt/spark-2.4.3"
```

Comment:
This is because `spark-submit` might expect this variable to be set. Modern versions of Spark have `spark-submit` that not require this variable to be set I think, but just to be safe I set it anyway.